hooks/cascading-scans/Makefile

```diff
@@ -8,10 +8,14 @@
 include_guard = set
 hook = cascading-scans
 
+test_files = [hook.test.js, kube.test.js]
+
 include ../../hooks.mk
 
+.PHONY: test-2
 test-2: | clean-integration-tests unit-tests docker-build docker-export kind-import deploy deploy-test-deps-2 integration-tests-2
```

Comment:
What does the `|` here? Never seen this before and I'm too stupid to google it...

Reply:
* https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html 
* https://stackoverflow.com/questions/58039810/makefiles-what-is-an-order-only-prerequisite

Reply:
Second question: What does the 2 mean in the target name? We should avoid magic numbers in names.